src/inverseRobot.js

```diff
@@ -4,10 +4,20 @@
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = Object.entries(robot).map(([a, b]) => [b, a]);
+
+  for (let i = 0; i < newRobot.length - 1; i++) {
+    if (newRobot[0][0] === newRobot[i + 1][0]) {
```

Comment:
You should not rely on order of keys in objects. They are not always sorted as expected. You can check my example from a screenshot. It is better to check whether such key is already present in an array 
![Screenshot 2023-02-27 at 17 48 36](https://user-images.githubusercontent.com/6526149/221611743-5e82cb6b-6a05-4f96-bc02-e965c2749576.png)

Reply:
Should I rebuild a "for" block?